cocos/ui/UIPageView.cpp

```diff
@@ -409,16 +469,16 @@ void PageView::doLayout()
     _doLayoutDirty = false;
 }
 
-void PageView::movePages(float offset)
+void PageView::movePages(Vec2 offset)
```

Comment:
I have restricted the touch to one direction at a time. In the touch moved method, if the direction is horizontal, then the the movement in y direction is ignored.
So it's ok here.